.github/workflows/csharp-tests.yml

```diff
@@ -84,9 +84,16 @@ jobs:
       run: cd bindings/csharp && dotnet restore
     - uses: actions/download-artifact@v4
       with:
-        pattern: 'ckzg-library-wrapper-${{ matrix.target.location }}-*'
-        path: bindings/csharp/Ckzg.Bindings/runtimes/${{ matrix.target.location }}/native
-        merge-multiple: true
+        pattern: ckzg-library-wrapper-*
+        path: bindings/csharp/Ckzg.Bindings/runtimes
+    - name: Move artifacts
+      working-directory: bindings/csharp/Ckzg.Bindings/runtimes
+      run: |
+        mv ckzg-library-wrapper-linux-arm64/ckzg.so linux-arm64/native/ckzg.so
+        mv ckzg-library-wrapper-linux-x64/ckzg.so linux-x64/native/ckzg.so
+        mv ckzg-library-wrapper-osx-arm64/ckzg.dylib osx-arm64/native/ckzg.dylib
+        mv ckzg-library-wrapper-osx-x64/ckzg.dylib osx-x64/native/ckzg.dylib
+        mv ckzg-library-wrapper-win-x64/ckzg.dll win-x64/native/ckzg.dll
```

Comment:
Since the checks have been removed from the `.csproj` file, we need to ensure that each artifact is present even though it is not used on the platform being run.